rasa/nlu/persistor.py

```diff
@@ -162,7 +162,10 @@ class GCSPersistor(Persistor):
     Fetches them when needed, instead of storing them on the local disk."""
 
     def __init__(self, bucket_name: Text) -> None:
-        from google.cloud import storage
+        """Initialise class with client and bucket."""
+        # there are no type hints in this repo for now
```

Comment:
To double-check I understand this correctly, if there's 3rd party code without type hints without type hints, mypy will complain?

Reply:
yes exactly. It's enabled by https://github.com/RasaHQ/rasa/blob/348697b7e43d5aa478aa3dc7ccd3fc2140facebc/setup.cfg#L43